ComposeTesting/dependencies.gradle

```diff
@@ -14,11 +14,11 @@ ext {
 
     // ComposeTesting library version code:
     // If you want to publish a new version, bump in one (1) the specific line(s)
-    composeTestingVersionCode = 2
+    composeTestingVersionCode = 3
```

Comment:
Do you think we should wait until the ComposeTesting update to bump the version? I'm ok with it either way, just want to see what you think!

Reply:
Yes, we should wait. The same as the sample PR.